config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rails secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: 2779215b126fa4ec497375b5c79bd7659064c17b27fe488eb41d89c466237d7a05d247bb75c366b69f4039c17b0362e9c468ccd94968de155b5c3c82e30613e5
```

Comment:
You'll want to put these in a git-ignored file. Figaro gem is often used which creates an application.yml file and puts it in your git-ignore.